tests/client/test_auth.py

```diff
@@ -152,7 +152,7 @@ async def async_auth_flow(
         yield request
 
 
-@pytest.mark.asyncio
+@pytest.mark.anyio
```

Comment:
is there a reason these were not marked `@pytest.mark.usefixtures("async_environment")` ?